digiwf-apps/pom.xml

```diff
@@ -7,7 +7,7 @@
   <parent>
     <groupId>io.muenchendigital.digiwf</groupId>
     <artifactId>digiwf-core</artifactId>
-    <version>0.17.10-SNAPSHOT</version>
+    <version>0.17.19-SNAPSHOT</version>
```

Comment:
Das passt nicht. Aktuell in dev ist 0.18.0

Reply:
@darenegade ist angepasst